conftest.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import pytest
+
+
+def pytest_collection_modifyitems(session, config, items):
```

Comment:
This just moved from `tests_integration/conftest.py` here to the root.  Its a better fit...